android-components/components/feature/push/src/test/java/mozilla/components/feature/push/AutoPushFeatureTest.kt

```diff
@@ -50,27 +44,14 @@ class AutoPushFeatureTest {
     @get:Rule
     val coroutinesTestRule = MainCoroutineRule()
 
-    private var lastVerified: Long
-        get() = preference(testContext).getLong(LAST_VERIFIED, System.currentTimeMillis())
-        set(value) = preference(testContext).edit().putLong(LAST_VERIFIED, value).apply()
-
-    private val connection: PushConnection = mock()
-
-    @Before
-    fun setup() {
-        whenever(connection.isInitialized()).thenReturn(true)
-    }
-
-    @After
-    fun shutdown() {
-        preference(testContext).edit().remove(LAST_VERIFIED).apply()
-    }
+    private val connection: PushManagerInterface = mock()
```

Comment:
Without looking too deeply into these tests yet, I'm wondering if this should be `TestPushConnection`

Reply:
Aaah `TestPushConnection` was unnecessary - got rid of it!